api-extractor.json

```diff
@@ -0,0 +1,364 @@
+         * prepend a folder token such as "<projectFolder>".
+         *
+         * Note: This setting will be ignored if "overrideTsconfig" is used.
+         *
+         * SUPPORTED TOKENS: <projectFolder>, <packageName>, <unscopedPackageName>
+         * DEFAULT VALUE: "<projectFolder>/tsconfig.json"
+         */
+        // "tsconfigFilePath": "<projectFolder>/tsconfig.json",
+        /**
+         * Provides a compiler configuration that will be used instead of reading the tsconfig.json file from disk.
+         * The object must conform to the TypeScript tsconfig schema:
+         *
+         * http://json.schemastore.org/tsconfig
+         *
+         * If omitted, then the tsconfig.json file will be read from the "projectFolder".
+         *
+         * DEFAULT VALUE: no overrideTsconfig section
+         */
+        // "overrideTsconfig": {
+        //   . . .
+        // }
+        /**
+         * This option causes the compiler to be invoked with the --skipLibCheck option. This option is not recommended
+         * and may cause API Extractor to produce incomplete or incorrect declarations, but it may be required when
+         * dependencies contain declarations that are incompatible with the TypeScript engine that API Extractor uses
+         * for its analysis.  Where possible, the underlying issue should be fixed rather than relying on skipLibCheck.
+         *
+         * DEFAULT VALUE: false
+         */
+        // "skipLibCheck": true,
+    },
+
+    /**
+     * Configures how the API report file (*.api.md) will be generated.
+     */
+    "apiReport": {
+        /**
+         * (REQUIRED) Whether to generate an API report.
+         */
+        "enabled": true
+
+        /**
+         * The filename for the API report files.  It will be combined with "reportFolder" or "reportTempFolder" to produce
+         * a full file path.
+         *
+         * The file extension should be ".api.md", and the string should not contain a path separator such as "\" or "/".
+         *
+         * SUPPORTED TOKENS: <packageName>, <unscopedPackageName>
+         * DEFAULT VALUE: "<unscopedPackageName>.api.md"
+         */
+        // "reportFileName": "<unscopedPackageName>.api.md",
+
+        /**
+         * Specifies the folder where the API report file is written.  The file name portion is determined by
+         * the "reportFileName" setting.
+         *
+         * The API report file is normally tracked by Git.  Changes to it can be used to trigger a branch policy,
+         * e.g. for an API review.
+         *
+         * The path is resolved relative to the folder of the config file that contains the setting; to change this,
+         * prepend a folder token such as "<projectFolder>".
+         *
+         * SUPPORTED TOKENS: <projectFolder>, <packageName>, <unscopedPackageName>
+         * DEFAULT VALUE: "<projectFolder>/etc/"
+         */
+        // "reportFolder": "<projectFolder>/etc/",

```

Comment:
"reportFolder": "<projectFolder>/docs/"
->
./docs/oidc-client-ts.api.md

see other comment for more context...

Reply:
Sounds good, and makes sense.

Also, is the Docusaurus site building for you since the merge of all of those dependabot PRs? Both `docusaurus-plugin-typedoc` and `typedoc-plugin-markdown` needed updates to support Typedoc 0.22. The webpack build inconsistently reports between being unable to find `api/index.md` or that the sidebar can't render due to `api/index` being an unknown document name.

Reply:
lets fix that i a separate merge request